### DISCUSSION TITLE
List Organisation Repo Branches

### DISCUSSION THREAD

**Discussion by ghost** [community/community#40264] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

May be a silly question but I can't find documentation for list branches of an organisation repo. 
I can see it for listing branches and listing orgs and I am able to list the branches in the org, but not the branches of a specified repo within the org.

What am I missing? Is it right in front of me..?

**Comment by ossanna16** [comment]
Hi there @mcgowan23 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
